packages/livekit-server-sdk/src/AgentDispatchClient.ts

```diff
@@ -29,10 +33,15 @@ export class AgentDispatchClient extends ServiceBase {
    * @param host - hostname including protocol. i.e. 'https://<project>.livekit.cloud'
    * @param apiKey - API Key, can be set in env var LIVEKIT_API_KEY
    * @param secret - API Secret, can be set in env var LIVEKIT_API_SECRET
+   * @param options - client options
+   * @param options.requestTimeout - optional timeout, in seconds, for all server requests
```

Comment:
Instead of describing the usage here, could you do it just above the field definition in the `ClientOptions` type? 

Would also be nice if we could share the `ClientOptions` type across different implementations instead of redefining it in each client module.